themes/classic/_dev/css/modules/customer.scss

```diff
@@ -1,9 +1,19 @@
-.page-authentication {
-  .no-account {
-    text-align: center;
-    font-size: 1.25rem;
-    margin: 1rem 0;
+.table-labeled {
+  th {
```

Comment:
In this case in particular, you should simply use `text-center` on the template.